website/README.md

```diff
@@ -0,0 +1,17 @@
+# MetalLB Documentation Website
```

Comment:
I'd put this under the "contributing" section of the website, with the suggestion that any new feature should be documented

Reply:
I think the discoverability of an inner README is quite low :-)

Now that I think about it, I'd rather add yet another inv command (something like `inv preview-website`) that downloads hugo locally and previews the website, so a new contributor won't even have the burden of finding the right version, checking how to build it and to run it.

And we don't need to do it here, I'll file a separate issue but I think it's the right path. On the doc page we can just say "run inv preview" and check if your changes are correct.

Reply:
I think it's better that way.